service-maintenance/internal/server/handlers/ual_maintenance.go

```diff
@@ -0,0 +1,51 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/rs/zerolog/log"
+)
+
+func UseALPAEnglishMaintenanceHandler() http.HandlerFunc {
```

Comment:
I wonder if these could just be a generic static page handler maker that takes in the Name of the page to log and then the name of the template

Reply:
e.g 

handlers.StaticPageHander("LPA English Maintenance", "ual_en_maintenance.page.gohtml")

Reply:
I was thinking the same, there's a lot of repetition. I'm proposing to change that when I come to pass in content from AWS Parameter Store or some other storage area, then it would be title, template and location of dynamically loaded content. I'd create a new object alongside the existing then refactor these functions out.